docs(spark-setup): improve namespace caveat description (#5952)
fishtown-analytics/docs.getdbt.com: website/docs/docs/core/connect-data-platform/spark-setup.md

## Caveats

> When facing difficulties you can run `poetry run dbt debug --log-level=debug`. The logs are persisted at `logs/dbt.log`.

Reviewer: ```suggestion
When facing difficulties, run `poetry run dbt debug --log-level=debug`. The logs are saved at `logs/dbt.log`.
```

### Default namespace with Thrift connection method

> A namespace named `default` is required to exist in Spark when connecting via Thrift for dbt to run metadata queries in. You can use Spark's `pyspark` and run `spark.sql("SHOW NAMESPACES").show()` to see the available namespaces and create the required namespace by running `spark.sql("CREATE NAMESPACE default").show()`.

Reviewer: ```suggestion
To run metadata queries in dbt, you need to have a namespace named `default` in Spark when connecting with Thrift. You can check available namespaces by using Spark's `pyspark` and running `spark.sql("SHOW NAMESPACES").show()`. If the default namespace doesn't exist, create it by running `spark.sql("CREATE NAMESPACE default").show()`.
```

> If there's a network connection issue instead, your logs will contain `Could not connect to any of [('127.0.0.1', 10000)]` (or similar).

Reviewer: ```suggestion
If there's a network connection issue, your logs will dispaly an error like `Could not connect to any of [('127.0.0.1', 10000)]` (or something similar).
```

Author: I think there is a typo in "dispaly"

Reviewer: good catch! will change this for you!
